### PR TITLE
sqlite3:insert added #:or-fail

### PR DESCRIPTION
**Pull request by Rscho314**
Another one adding the `INSERT OR FAIL` option for SQLite.

### CODE REVIEW

File: private/emit.rkt [hunk 1]
```diff
@@ -222,6 +222,11 @@
         [(statement:insert table columns source on-conflict)
          (case on-conflict
            [(#f) (J "INSERT INTO " (emit-insert* i))]
+           [(fail)
+            (case (send dialect insert/on-conflict-style)
+              [(sqlite3) (J "INSERT OR FAIL INTO " (emit-insert* i))]
+              [else (error 'emit-insert "ignore option not supported~a"
```
Comment (rmculpepper) [inline private/emit.rkt]: The error message should say "fail" instead of "ignore".

File: sql.scrbl [hunk 2]
```diff
@@ -864,7 +864,8 @@ clause may not also be used.
                (code:line #:columns column-ident ...)]
 
 [maybe-conflict (code:line)
-                (code:line #:or-ignore)]
+                (code:line #:or-ignore)
+		(code:line #:or-fail)]
```
Comment (rmculpepper) [inline sql.scrbl]: Please convert tabs to spaces.

**Review by rmculpepper — COMMENTED** [review]
Two small issues, but otherwise looks good. Thanks!

**Comment by Rscho314** [comment]
Hi Ryan,

Sorry for the super long delay! I'm struggling at work. I hopefully made the changes without messing up.

**Review by rmculpepper — APPROVED** [review]
Looks good, thanks!

**Pull request merged**